tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -79,7 +80,7 @@ Switch To New Tab
 Click On Searched Operator
     [Arguments]   ${operator}
      Search Installed Operator          ${operator}
-     Sleep   1
+     Wait Until Element is Visible     xpath=//a[@data-test-operator-row="${operator}"]    timeout=10
      Click Element       xpath=//a[@data-test-operator-row="${operator}"]
      Wait until page contains           Description           timeout=10
```

Comment:
Shouldn't this be split in two keywords? `Search for Operator` and `Click on Operator`?